cfme/web_ui/__init__.py

```diff
@@ -1280,6 +1280,7 @@ def _fill_form_list(form, values, action=None, action_always=False):
 
     """
     logger.info('Beginning to fill in form...')
+    sel.wait_for_ajax()
```

Comment:
Okay, it seems the problem was not in ajax and focus but some code quirks - host tests verified validation twice - and the extra validation was failing on upstream. Fixed and restarted tests